resources/metadata/transportRequestUploadRFC.yaml

```diff
@@ -0,0 +1,144 @@
+        type: jenkins
+        aliases:
+          - name: changeManagement/credentialsId
+    params:
+      - name: endpoint
+        type: string
+        mandatory: true
+        description: "The service endpoint"
+        aliases:
+          - name: changeManagement/endpoint
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: client
+        type: string
+        aliases:
+          - name: changeManagement/rfc/developmentClient
+        description: "AS ABAP client number"
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: instance
+        type: string
+        aliases:
+          - name: changeManagement/rfc/developmentInstance
+        description: "AS ABAP instance number"
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: username
+        type: string
+        mandatory: true
+        description: "The deploy user"
+        secret: true
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: password
+        type: string
+        mandatory: true
+        description: "The password for the deploy user"
+        secret: true
+        scope:
+          - PARAMETERS
+      - name: applicationName
+        type: string
+        mandatory: true
+        description: "The name of the application."
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: applicationDescription
+        type: string
+        mandatory: false
+        description: "The description of the application."
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: abapPackage
+        type: string
+        mandatory: true
+        description: "The ABAP package name of your application"
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: applicationUrl
+        type: string
+        mandatory: true
+        description: "The URL where to find the UI5 package to upload to the transport request"
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: abapPackage
+        type: string
+        mandatory: true
+        description: "The ABAP package name of your application"
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: codePage
+        type: string
+        default: "UTF-8"
+        description: "The Code page"
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+      - name: acceptUnixStyleLineEndings
+        type: bool
+        default: true
+        description: "If unix style line endings should be accepted"
```

Comment:
In that case they are not accepted, I suppose.

Reply:
It seems that for (older?) abap systems "`\n`" is just a control character (and not allowed as line ending?). But whatever the command uploads into the cts will contain linefeeds, right? does it make sense to set it to `false`?

Reply:
Don't know any details on ABAP side. Don't know if it makes sense to set it to `false`. I it just a parameters exposed by the backend. We have the choice to silently set a value, in that case we are responsible in case the value does not match somewhere or we can forward the parameter.

Reply:
Anybody, who can tell us?

Reply:
@radsoulbeard : do we have an ABAP contact for the question here?